<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>comic_widget</class>
 <widget class="QDialog" name="comic_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>8</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="image_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>202</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="image_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>202</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_box_status">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>236</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="info_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>202</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_down">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_like">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>♥ Like ♥</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_star">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Star Score ★</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_comment">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="btn_layout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
